src/php/docker/i386/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM i386/php:7.2
```

Comment:
Let's add this via the `templates/src/php/docker` directory. Also let's add this to the `src/php/bin/[build|run]_all_docker_images.sh`.

Reply:
comment addressed